community.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Community
+permalink: /community/
+header_image: contact
+---
+
+# Community
+Nuts is een community initiatief, waarbij we samen bouwen aan een decentraal communicatienetwerk voor de zorg. Voor een effectieve samenwerking is het essentieel dat we elkaar kunnen ontmoeten en informeren over de voortgang. 
```

Comment:
community-initiatief (in principe één woord, maar dan wordt het slecht leesbaar, dus dan gebruik je een koppelteken)